guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleAuthenticationProvider.java

```diff
@@ -260,6 +260,23 @@ public UserContext updateUserContext(UserContext context,
         
     }
 
+    @Override
+    public UserContext decorate(UserContext context,
+            AuthenticatedUser authenticatedUser, Credentials credentials)
+            throws GuacamoleException {
+
+        // Simply return the given context, decorating nothing
+        return context;
+
+    }
+
+    @Override
+    public UserContext redecorate(UserContext decorated, UserContext context,
+            AuthenticatedUser authenticatedUser, Credentials credentials)
+            throws GuacamoleException {
+        return decorate(context, authenticatedUser, credentials);
```

Comment:
Any particular reason in this instance you chose to pass up to the `decorate()` method rather than just returning the context?  In the extension modules you just `return context`, instead - why the difference?

Reply:
Mainly because this is part of the public API and not the internal workings of an extension. I chose to do things this way so that the class remains compliant with the expectations of `decorate()` / `redecorate()` if `decorate()` is overridden while `redecorate()` is not.

Reply:
:+1: